plugins/spectate.cpp

```diff
@@ -66,23 +145,77 @@ DFhackCExport command_result plugin_enable(color_ostream &out, bool enable) {
         EM::registerListener(EventType::JOB_STARTED, start, plugin_self);
         EM::registerListener(EventType::JOB_COMPLETED, complete, plugin_self);
     } else if (!enable && enabled) {
+        // warp 8, engage!
         out.print("Spectate mode disabled!\n");
         EM::unregisterAll(plugin_self);
         job_tracker.clear();
         freq.clear();
     }
     enabled = enable;
-    return CR_OK;
+    return DFHack::CR_OK;
 }
 
 command_result spectate (color_ostream &out, std::vector <std::string> & parameters) {
-    return plugin_enable(out, !enabled);
+    // todo: parse parameters
+    if(!parameters.empty()) {
+        if (parameters[0] == "disable") {
+            return plugin_enable(out, false);
+        } else if (parameters[0] == "enable") {
+            return plugin_enable(out, true);
+        } else if (parameters[0] == "godmode") {
+            out.print("todo?\n"); // todo: adventure as deity?
```

Comment:
Could consider commenting this entire block out (if you want to keep it around) so that it can't be invoked until it does something